tests/integration/test_charm.py

```diff
@@ -140,21 +120,33 @@ async def test_tracing(ops_test: OpsTest):
 
     assert unit_relation_data
 
+async def test_integrate_blackbox(ops_test: OpsTest):
+    await ops_test.model.deploy("blackbox-exporter-k8s", "blackbox", channel="latest/edge", trust=True)
```

Comment:
the channel "latest/edge" is picked for a reason? are we sure this won't bring instability in the tests?

Reply:
ah right, I had picked it for the library, but the library is copied in every charm. good catch, removing it.